source/releasenotes/2025-01-05-new-relic-agent.md

```diff
@@ -0,0 +1,11 @@
+---
+title: New Relic PHP agent v11.8.0.22 now available
+published_date: "2025-05-01"
+categories: [infrastructure, performance, tools-apis]
+---
+
+The New Relic PHP agent has been updated to the latest version, v11.8.0.22, for all eligible versions of PHP. Support for PHP 7.0 and 7.1 was removed in version 11.0, so the New Relic PHP agent for those versions remains at v10.22.0.12.
```

Comment:
@kporras07 Perhaps worth a quick note about how this will be rolled out (aka may take a couple days for all sites to get the update)

Reply:
Do you have any text suggestion? TBH, I mostly duplicated an existing note. This suggestion is great! :)

Reply:
LGTM!